deployments/ansible/roles/collector/defaults/main.yml

```diff
@@ -1,16 +1,18 @@
 ---
 # defaults file for splunk-otel-collector role vars
-
+# Note: To modify vars based upon ansible_os_family  change the values of vars in tasks/vars.yml
 # splunk_access_token is REQUIRED variable
 splunk_access_token: ""
 
 splunk_realm: us0
 
 splunk_otel_collector_version: latest
 
-# Set this variable to /etc/otel/collector/gateway_config.yaml in order to
-# deploy splunk-otel-collector in gateway mode
-splunk_otel_collector_config: /etc/otel/collector/agent_config.yaml
+# Set this variable to /etc/otel/collector/gateway_config.yaml on linux,
+# "{{ansible_env.ProgramFiles}}/Splunk/OpenTelemetry Collector/gateway_config.yaml" on Windows
```

Comment:
Shouldn't it be `%ProgramFiles%/Splunk/OpenTelemetry Collector/gateway_config.yaml`? I don't think that jinja template evaluation will be applied on user input

Reply:
If we use ` %ProgramFiles%/Splunk/OpenTelemetry Collector/gateway_config.yaml ` Windows reading this value as a literal, we need this value to be assigned as ENV var for SPLUNK_CONFIG. So, I have used 
` {{ansible_env.ProgramFiles}}/Splunk/OpenTelemetry Collector/gateway_config.yaml` to resolve that issue.

Reply:
But if user uses `{{ansible_env.ProgramFiles}}/Splunk/OpenTelemetry Collector/gateway_config.yaml` value as is in `splunk_otel_collector_config` ansible will likely use the value as is without evaluating `ansible_env.ProgramFiles`, so it won't work. We probably should specify something like `Program Files\Splunk\OpenTelemetry Collector\gateway_config.yaml`